test/WebSites/NswagClientExample/Controllers/SystemTextJsonAnimalsController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace NSwagClientExample.Controllers
+{
+#if NET7_0_OR_GREATER
+    [ApiController]
+    [Route("[controller]")]
+    public class SystemTextJsonAnimalsController : ControllerBase
+    {
+        [HttpPost]
```

Comment:
Could we add `[Produces(...)]` here for JSON? Then we shouldn't(?) get three different content-types that are all the basically the same in the responses in the snapshot.

Reply:
added it.